zathura/database-sqlite.c

```diff
@@ -148,7 +148,7 @@ sqlite_db_init(ZathuraSQLDatabase* db, const char* path)
     "scale FLOAT,"
     "rotation INTEGER,"
     "pages_per_row INTEGER,"
-    "first_page_column INTEGER,"
+    "first_page_column TEXT,"
```

Comment:
This change needs code to alter old databases similar to the other `ALTER` statements we have to migrate old database layouts.